tests/instances.tcl

```diff
@@ -252,6 +264,8 @@ proc parse_options {} {
             set ::simulate_error 1
         } elseif {$opt eq {--valgrind}} {
             set ::valgrind 1
+        } elseif {$opt eq {--sanitizer}} {
+            set ::sanitizer 1
```

Comment:
why do we need this optional? why not have that always?

Reply:
Made it inline with valgrind setting, it checks logs for sanitizer errors when built with sanitizer only. Otherwise, it'll search logs unnecessarily. I can make it default if you are okay with that.

Reply:
ok, got rid of the argument and made it look for sanitizer errors by default